examples/from_file.rb

```diff
@@ -0,0 +1,4 @@
+require_relative '../lib/ffprober'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.